README.md

```diff
@@ -97,6 +97,7 @@ Name                      | Description
 `ZCBOR_ASSERTS`           | Enable asserts (`zcbor_assert()`). When they fail, the assert statements instruct the current function to return a `ZCBOR_ERR_ASSERTION` error. If `ZCBOR_VERBOSE` is enabled, a message is printed.
 `ZCBOR_STOP_ON_ERROR`     | Enable the `stop_on_error` functionality. This makes all functions abort their execution if called when an error has already happened.
 `ZCBOR_BIG_ENDIAN`        | All decoded values are returned as big-endian. The default is little-endian.
+`ZCBOR_MAP_SEARCH_FLAGS`  | Applies to decoding of unordered maps. When enabled, a flag is kept for each element in an array, ensuring it is not processed twice. If disabled, a count is kept for map as a whole. Enabling increases code size and memory usage, and requires the state variable to possess the memory necessary for the flags.
```

Comment:
The benefit is

> When enabled, a flag is kept for each element in an array, ensuring it is not processed twice

This means that when you search for something, you won't find the same element again since zcbor_unordered_map_search() checks the flag for that element.